nimble/controller/src/ble_ll_iso_big.c

```diff
@@ -160,6 +164,58 @@ static uint8_t bis_pool_free = BIS_POOL_SIZE;
 static struct ble_ll_iso_big *big_pending;
 static struct ble_ll_iso_big *big_active;
 
+struct ble_ll_iso_bis *
+ble_ll_iso_big_find_bis_by_handle(uint16_t conn_handle)
+{
+    struct ble_ll_iso_bis *bis;
+    uint8_t bis_idx;
+
+    if (!BLE_LL_CONN_HANDLE_IS_BIS(conn_handle)) {
+        return NULL;
+    }
+
+    bis_idx = BLE_LL_CONN_HANDLE_IDX(conn_handle);
+
+    if (bis_idx >= BIS_POOL_SIZE) {
+        return NULL;
+    }
+
+    bis = &bis_pool[bis_idx];
+    if (!bis->big) {
+        return NULL;
+    }
+
+    return bis;
+}
+
+struct ble_ll_isoal_mux *
+ble_ll_iso_big_find_mux_by_handle(uint16_t conn_handle)
+{
+    struct ble_ll_iso_bis *bis;
+
+    bis = ble_ll_iso_big_find_bis_by_handle(conn_handle);
+    if (bis) {
+        return &bis->mux;
+    }
+
+    return NULL;
+}
+
+int
+ble_ll_iso_big_last_tx_timestamp_get(struct ble_ll_iso_bis *bis,
+                                     uint16_t *packet_seq_num, uint32_t *timestamp)
+{
+    struct ble_ll_iso_big *big;
+
+    big = bis->big;
+
+    *packet_seq_num = big->bis_counter;
+    *timestamp = (uint64_t)big->event_start * 1000000 / 32768 +
```

Comment:
some comment would be helpful here (unless we have some macro that would cover this)